packages/veritone-react-common/src/components/LogoDetectionEngineOutput/index.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import classNames from 'classnames';
+import IconButton from 'material-ui/IconButton';
+import ZoomOutMap from 'material-ui-icons/ZoomOutMap';
+import Select from 'material-ui/Select';
+import { MenuItem } from 'material-ui/Menu';
+import EngineOutputHeader from '../EngineOutputHeader';
+import PillButton from '../Parts/Buttons/PillButton';
+import { msToReadableString } from '../../helpers/time'
+import styles from './styles.scss';
+
+export default class LogoDetectionEngineOutput extends Component {
+	static propTypes = {
+		label: PropTypes.string,
+		data: PropTypes.arrayOf(Object),
+		mediaPlayerTime: PropTypes.number,
+		className: PropTypes.string,
+		headerClassName: PropTypes.string,
+		bodyClassName: PropTypes.string,
+		itemClassName: PropTypes.string,
+		itemLabelClassName: PropTypes.string,
+		itemInfoClassName: PropTypes.string,
+		onItemSelected: PropTypes.func,
+		onEngineSelected: PropTypes.func,
+		onExpandViewClicked: PropTypes.func
+	};
+
+	static defaultProps = {
+		label: 'Logo Recognition',
+		data: [],
+		mediaPlayerTime: -1
+	};
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			isExpanded: false
+		};
+
+		this.onEngineChanged = this.onEngineChanged.bind(this);
+		this.onExpandViewClicked = this.onExpandViewClicked.bind(this);
+	}
+
+	//================================================================================
+	//=============================Interaction Functions==============================
+	//================================================================================
+	onItemSelected = item => (event) => {
+		let itemInfo = {
+			item: item,
+			sourceEngineId: this.state.sourceEngineId || this.props.data[0].sourceEngineId
+		}
+
+		if (this.props.onItemSelected) this.props.onItemSelected(itemInfo);
+	}
+
+	onEngineChanged(event) {
+		let selectedID = event.target.value;
+		this.setState({
+			sourceEngineId: selectedID
+		});
+
+		if (this.props.onEngineSelected) this.props.onEngineSelected(selectedID);
```

Comment:
Usual Veritone code style is
```
if () {
  // always drop onto a second line
}
```
...unless I've been out of touch for too long and someone like @olslash or @ChrisPelletier corrects me.

Please correct all over.  I'm actually surprised prettier doesn't enforce something like that.